tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
-	"include": [],
+	"include": ["src/**/*"],
 
 	"compilerOptions": {
 		"rootDir": "src",
 
 		// target node v8+ (https://node.green/)
```

Comment:
Hi all, sorry a question: if target Node.js version here is 14, wouldn't it be good to update even following settings to ES2020 ? 
Sorry another question: when Svelte 4.x will be released probably minimum Node.js version supported will be 16.x (and in maintenance mode), so why not align to current LTS release (it could become 18 LTS), as seen [here](https://github.com/nodejs/Release) ?
Thanks and sorry for questions here. Regards